### PR TITLE
Chore unify logging of client keys

### PR DESCRIPTION
**Pull request by pawelkisielewicz**

### CODE REVIEW

File: concent_api/utils/logging.py [hunk 1]
```diff
@@ -1,112 +1,121 @@
...
-    ))
-    logger.debug('A message has been received in `send/` -- MESSAGE: {}'.format(
-        message,
-    ))
+def replace_key_to_unavailable_instead_of_none(log_function):
+    def wrap(*args):
+        if args[1] is None:
+            args_list = list(args)
+            args_list[1] = 'UNAVAILABLE'
+            args = log_function(*tuple(args_list))
```
Comment (rwrzesien) [inline concent_api/utils/logging.py]: `log_function` will not be called when `if args[1] is None:` condition is not met.

File: concent_api/utils/logging.py [hunk 2]
```diff
@@ -1,112 +1,121 @@
...
-        client_public_key,
-    ))
-    logger.debug('A message has been received in `send/` -- MESSAGE: {}'.format(
-        message,
-    ))
+def replace_key_to_unavailable_instead_of_none(log_function):
+    def wrap(*args):
+        if args[1] is None:
+            args_list = list(args)
+            args_list[1] = 'UNAVAILABLE'
```
Comment (rwrzesien) [inline concent_api/utils/logging.py]: Why not just 
`args[1] = args[1] if args[1] is not None else 'UNAVAILABLE'`

File: concent_api/utils/logging.py [hunk 3]
```diff
@@ -115,52 +124,55 @@ def log_message_not_allowed(endpoint: str, method: str, client_public_key: str):
 
 
 def log_subtask_stored(
-    task_id:              str,
-    subtask_id:           str,
-    state:                str,
-    provider_public_key:  str,
-    requestor_public_key: str,
-    next_deadline:        int          = None,
+        task_id: str,
```
Comment (rwrzesien) [inline concent_api/utils/logging.py]: Too much indentation here.

File: concent_api/utils/logging.py [hunk 4]
```diff
@@ -115,52 +124,55 @@ def log_message_not_allowed(endpoint: str, method: str, client_public_key: str):
...
+            provider_public_key,
+            requestor_public_key,
+        ))
 
 
 def log_stored_message_added_to_subtask(
-    subtask_id:           str,
-    state:                str,
-    stored_message:  Message,
+        subtask_id: str,
```
Comment (rwrzesien) [inline concent_api/utils/logging.py]: Same here.

**Comment by rwrzesien** [comment]
@kbeker @pawelkisielewicz 
> Isn't this PR the same as #345 ?

I am a little bit confused too.

**Comment by cameel** [comment]
I think that
- #345 solves #173
- #346 (this pull request) solves #337

but @pawelkisielewicz just mistakenly linked #345 to #337 and did not link this pull request to anything.

**Comment by cameel** [comment]
> And please revert pylint auto formatting, because it's hard to see which changes resolves actual issue.

Reformatting is fine by I'd say that it should always be in a separate commit. It really obscures the actual changes.

**Pull request closed without merging**